.travis.yml

```diff
@@ -12,7 +12,7 @@ install: npm install
 script:
   - npm test
   - nsp check
```

Comment:
Maybe update this to `nsp check; exit 0`, since snyk will handle nsp checks on PR's, and we don't want travis failing our builds for existing vulnerabilities.